lib/src/components/NxNexusPageHeader/NxNexusPageHeader.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import classnames from 'classnames';
+
+import { HeaderLinkProps, Props, propTypes } from './types';
+
+export { Props };
+
+import './NxNexusPageHeader.scss';
+
+const logoImg = require('../../assets/img/logo_nexus_generic.svg');
+
+function HeaderLink({ name, href, current }: HeaderLinkProps) {
+  const classes = classnames('nx-page-header__link', {
+    'nx-page-header__link--current': current
+  });
+
+  return (
+    <a className={classes} href={href} data-text={name}>{name}</a>
+  );
+}
+
+// visible for testing
+export { HeaderLinkProps, HeaderLink };
+
+export default function NxNexusPageHeader({ links, homeLink, productInfo, children }: Props) {
```

Comment:
There's a lot of code duplication between this and `NxPageHeader`.  Do you want me to try to work with you on reducing that?

Reply:
> There's a lot of code duplication between this and `NxPageHeader`. Do you want me to try to work with you on reducing that?

Ross' deduce branch merged here: https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/4d77efee82d382db88ba0ea55c982c4214a3c300